frontend/docs/.vitepress/theme/custom.scss

```diff
@@ -0,0 +1,3 @@
+:root:not(.dark) .plugin-tabs--content div[class*="language-"] {
+    background-color: inherit !important;
```

Comment:
Why did this need to get added? Dont our docs already have some theme?

Reply:
If you see the `.plugin-tabs--content` part of the selector that style was overriding the background color in code blocks. I wanted to make code in tabs match code outside of tabs.